src/core/Akka.Persistence/JournalProtocol.cs

```diff
@@ -22,6 +22,16 @@ public interface IJournalMessage : IPersistenceMessage { }
     /// </summary>
     public interface IJournalRequest : IJournalMessage { }
 
+    /// <summary>
+    /// Marker for batched write operations
+    /// </summary>
+    public interface IJournalWrite : IJournalRequest { }
```

Comment:
Let's remove this and try to avoid making the `BatchingJournal`'s problem all of Akka.Persistence's problem.